### DISCUSSION TITLE
Note skinning

### DISCUSSION THREAD

**Discussion by rovqn** [ppy/osu#30569] [Ideas]
I feel like osu!mania lazer skin editor lacks a lot of the customizability that stable has to offer, one of the main things being changing the shape and or color of the incoming circles and differenciating between short and long notes. Also be able to adjust "Hit Position", "CollumnWidth" and "ColumnSpacing" as named in stable's skin.ini

**Comment by SevenChords** [comment]
A feature to adjust the ColumnSpacing from stables skin.ini in lazer would definitely improve readability for higher key count modes in mania
